### PR TITLE
feat(multicast.md): add examples

### PR DESCRIPTION
**Pull request by NyaGarcia**

### CODE REVIEW

File: scully.rxjs-docs.config.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { baseHrefRewrite } from '@scullyio/scully-plugin-base-href-rewrite';
 
-setPluginConfig(baseHrefRewrite, { href: '/rxjs-docu/' });
+/* setPluginConfig(baseHrefRewrite, { href: '/rxjs-docu/' }); */
 export const config: Partial<ScullyConfig> = {
   projectRoot: './src',
   projectName: 'rxjs-docs',
   outDir: './dist/static',
-  defaultPostRenderers: [baseHrefRewrite],
+  /* defaultPostRenderers: [baseHrefRewrite], */
```
Comment (Caballerog) [inline scully.rxjs-docs.config.ts:9]: This comment shouldn't be applied :-)

File: src/assets/scully-routes.json [hunk 2]
```diff
@@ -1 +1 @@
...
...dowTime.md"},{"route":"/operators/transformation/windowToggle","sourceFile":"windowToggle.md"},{"route":"/operators/transformation/windowWhen","sourceFile":"windowWhen.md"},{"route":"/operators/utility/delay","sourceFile":"delay.md"},{"route":"/operators/utility/delayWhen","sourceFile":"delayWhen.md"},{"route":"/operators/utility/dematerialize","sourceFile":"dematerialize.md"},{"route":"/operators/utility/finalize","sourceFile":"finalize.md"},{"route":"/operators/utility/observeOn","sourceFile":"observeOn.md"},{"route":"/operators/utility/repeat","sourceFile":"repeat.md"},{"route":"/operators/utility/repeatWhen","sourceFile":"repeatWhen.md"},{"route":"/operators/utility/subscribeOn","sourceFile":"subscribeOn.md"},{"route":"/operators/utility/tap","sourceFile":"tap.md"},{"route":"/operators/utility/timeInterval","sourceFile":"timeInterval.md"},{"route":"/operators/utility/timeout","sourceFile":"timeout.md"},{"route":"/operators/utility/timeoutWith","sourceFile":"timeoutWith.md"},{"route":"/version-6/deprecations","sourceFile":"deprecations.md"},{"route":"/version-6/migration","sourceFile":"migration.md"},{"route":"/version-6/pipeable-operators","sourceFile":"pipeable-operators.md"}]
```
Comment (Caballerog) [inline src/assets/scully-routes.json:1]: This file must to be discard because it is generated by the CI

**Review by Caballerog — CHANGES_REQUESTED** [review]

File: scully.rxjs-docs.config.ts (hunk 1 above)
Comment (NyaGarcia) [reply scully.rxjs-docs.config.ts:9]: Sh*t, that should not have been committed. My mistake. Fixing it now.

File: src/assets/scully-routes.json (hunk 2 above)
Comment (NyaGarcia) [reply src/assets/scully-routes.json:1]: That shouldn't have been committed either.

**Pull request merged**